execution_engine/src/storage/global_state/db.rs

```diff
@@ -267,7 +266,7 @@ impl DbGlobalState {
 
 fn memoized_find_missing_descendants(
     value_bytes: Bytes,
-    rocksdb: RocksDbStore,
+    rocksdb_store: &RocksDbTrieStore,
```

Comment:
I don't think we have to include the type of the DB in the name - maybe `trie_store` is enough.

Reply:
Done in [6c20f4a](https://github.com/casper-network/casper-node/pull/2999/commits/6c20f4a0f076a2a375dc7d0f332871c085825545)